src/fastmcp/utilities/skills.py

```
@@ -0,0 +1,250 @@
+    manifest_uri = f"skill://{skill_name}/_manifest"
+    result = await client.read_resource(manifest_uri)
+
+    if not result:
+        raise ValueError(f"Could not read manifest for skill: {skill_name}")
+
+    content = result[0]
+    if hasattr(content, "text"):
+        try:
+            manifest_data = json.loads(content.text)
+        except json.JSONDecodeError as e:
+            raise ValueError(f"Invalid manifest JSON for skill: {skill_name}") from e
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)